src/main/java/org/springframework/integration/aws/outbound/SnsMessageHandler.java

```diff
@@ -120,18 +120,50 @@ public void setSubjectExpression(Expression subjectExpression) {
 		this.subjectExpression = subjectExpression;
 	}
 
+	/**
+	 * A fixed message-group ID to be set for messages sent to an SNS FIFO topic
+	 * from this handler.
+	 * Equivalent to calling {{@link #setMessageGroupIdExpression(Expression)} with
+	 * a literal string expression.
+	 *
+	 * @param messageGroupId the group ID to be used for all messages sent from this handler
+	 *
+	 * @since 2.5.3
+	 */
 	public void setMessageGroupId(String messageGroupId) {
-	    Assert.hasText(messageGroupId, "messageGroupId must not be empty.");
-	    this.messageGroupIdExpression = new LiteralExpression(messageGroupId);
+		Assert.hasText(messageGroupId, "messageGroupId must not be empty.");
+		this.messageGroupIdExpression = new LiteralExpression(messageGroupId);
 	}
 
+
+	/**
+	 * The {@link Expression} to determine the
+	 * <a href="https://docs.aws.amazon.com/sns/latest/dg/fifo-message-grouping.html">message group</a>
+	 * for messages sent to an SNS FIFO topic from this handler.
+	 *
+	 * @param messageGroupIdExpression the {@link Expression} to produce the message-group ID
+	 *
```

Comment:
No blank lines in the method Javadocs, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting